src/main/kotlin/no/nav/hjelpemidler/soknad/mottak/db/SoknadStore.kt

```diff
@@ -52,6 +58,35 @@ internal class SoknadStorePostgres(private val ds: DataSource) : SoknadStore {
         }
     }
 
+    override fun hentSoknadData(soknadsId: UUID): SoknadData? {
```

Comment:
joark-sink trenger søknaden for å generere og arkivere pdf. Ved FULLMAKT kjem allerede søknaden i meldinga frå hm-soknad-api, men for brukergodkjenning så må vi hente frå databasen

Reply:
hva ikke hele poenget med RnR meldinger at ting forblir løstkoblet? å unngå API kaller, osv? Hadde det ikke blitt bra om meldingen inneholder alt som trengs?

Reply:
Er enig i at det er meir RnR måten å gjere det på. I dette tilfellet så betyr det at `hm-brukerbekreftelse` må hente heile søknaden og sende vidare når bruker har bekrefta. Så det vil kreve litt større endringar. Vi kan diskutere det på møtet i morgon og finne ut om det er måten vi vil gjere det på.